docs/how-to/how-to-build-and-test-ccv0.md

```diff
@@ -333,6 +333,9 @@ to create containers from these images using `crictl`:
 the guest agent code has been implemented, the Kata confidential-containers build needs to be run with 
 `export SKOPEO=yes` as documented [above](#basic-script-set-up-and-optional-environment-variables) in order to use the
 image signature verification function.
+
+- The policy file, along with the public GPG key and signatures file, will need to be copied over before the pod is created by using the `copy_files_to_guest` function in ccv0.sh. To do this, you should run `~/ccv0.sh copy_files_to_guest` before creating the sandbox pod in the steps below.
```

Comment:
The preferred style for documentation is to keep the lines <=120 chars long in order to make it easier to see diff edits, so if you could add some line breaks in that would be great